template/build/dev-server.js

```diff
@@ -43,7 +45,7 @@ compiler.plugin('compilation', function (compilation) {
 
 // proxy api requests
 Object.keys(proxyTable).forEach(function (context) {
-  var options = proxyTable[context]
+  const options = proxyTable[context]
```

Comment:
`let` instead of `const`
https://github.com/vuejs-templates/webpack/pull/965

Reply:
Done with rebase.